### PR TITLE
Update to graph-cli / graph-ts 0.5.0

### PR DESCRIPTION
**Pull request by Jannis**
The title says it all.

### CODE REVIEW

File: src/ad-networks.ts [hunk 1]
```diff
@@ -20,6 +17,7 @@ import { Application, Challenge, User, Withdrawal } from './types/schema'
 
 // Respond to application white listed events
 export function applicationWhitelist(event: _ApplicationWhitelisted): void {
+  let foo = new Bytes()
```
Comment (leodasvacas) [inline src/ad-networks.ts]: What's this doing?

Comment (Jannis) [reply src/ad-networks.ts]: Haha, erm, I'll remove this.

Comment (Jannis) [reply src/ad-networks.ts]: I think that was from testing removing the empty constructors. 😉

**Comment by Jannis** [comment]
@leodasvacas Updated.

**Pull request merged**